app/components/remove_markdown/index.tsx

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Parser} from 'commonmark';
+import Renderer from 'commonmark-react-renderer';
+import React, {ReactElement, useRef} from 'react';
+import {Text} from 'react-native';
+
+type Props = {
+    renderSoftBreak?: () => string;
+    value: string;
+};
+
+const RemoveMarkdown = ({renderSoftBreak, value}: Props) => {
+    const renderText = ({literal}: {literal: string}) => {
+        return <Text>{literal}</Text>;
+    };
+
+    const renderNull = () => {
+        return null;
+    };
+
+    const createRenderer = () => {
+        return new Renderer({
+            renderers: {
+                text: renderText,
+
+                emph: Renderer.forwardChildren,
+                strong: Renderer.forwardChildren,
+                del: Renderer.forwardChildren,
+                code: Renderer.forwardChildren,
+                link: Renderer.forwardChildren,
+                image: renderNull,
+                atMention: Renderer.forwardChildren,
+                channelLink: Renderer.forwardChildren,
+                emoji: renderNull,
+                hashtag: Renderer.forwardChildren,
+
+                paragraph: Renderer.forwardChildren,
+                heading: Renderer.forwardChildren,
+                codeBlock: renderNull,
+                blockQuote: renderNull,
+
+                list: renderNull,
+                item: renderNull,
+
+                hardBreak: renderNull,
```

Comment:
this should also render a break, you can use the same function as softBreak if you want